q2_types/feature_data/_transformer.py

```diff
@@ -797,3 +798,22 @@ def _227(ff: BLAST6Format) -> qiime2.Metadata:
     # default int index but cast to a str and give it a name.
     data.index = pd.Index(data.index.astype(str), name='id')
     return qiime2.Metadata(data)
+
+
+@plugin.register_transformer
+def _228(ff: SequenceCharacteristicsFormat) -> pd.DataFrame:
+    return pd.read_csv(str(ff), sep="\t", index_col=0)
```

Comment:
```suggestion
    return pd.read_csv(str(ff), sep='\t', index_col=0)
```

Reply:
I know this is super nitpicky, but it really bothers me when single vs. double quotes aren't consistent (at least across individual files) 😝